src/Shared/Network/Connection.cs

```diff
@@ -126,6 +146,12 @@ private void OnReceive(IAsyncResult result)
 					return;
 				}
 
+				// When set, ignore packets until the client eventually terminates the connection.
+				if (this.IgnorePackets)
+				{
+					this.BeginReceive();
```

Comment:
You have to `return` if you don't want the packet to be handled, and I actually don't know what happens if you call `BeginReceive` twice =o What do you think about moving this further down where `HandlePacket` is called? You could wrap that call in an if checking for `IgnorePackets`, ignoring only the actual handling of the packets, and still get the parsing to make sure packets are valid.